libstuff/SRandom.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <random>
+using namespace std;
+
+// Random number generator class.
+class SRandom {
+private:
+    static mt19937_64* generator;
```

Comment:
Also, please move the `private:` section to the bottom, such that the `public:` section comes first -- this way you get to "the good stuff" faster when reading the file.